lib/enigma.rb

```diff
@@ -17,6 +17,14 @@ def encrypt(message, key = nil, date = nil)
     }
   end
 
+  def decrypt(message, key = nil, date = nil)
```

Comment:
decrypt is identical to encrypt, but uses a decrypt_msg_converter to change how to letter_converter shifts. I had to make a new decrypt_letter_converter to make this happen. How can I call on one method, msg_coverter, yet within that method call on 2 different letter_converters, for either the encrypt (+rotation) or decrypt(-rotation)?